### PR TITLE
Fixes the default separator for array fields in the database writer

### LINKED ISSUE #33: Documentation typo
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | all

#### Summary

This line:
 its elements will be translated as 'extra-field', where '-' is the character separator (default)

#### Current behavior

The default separator that is being used is an _ not a -

#### How to reproduce
Trying using the - as the default separator while writing a log to a table.
#### Expected behavior

That the docs include the correct default separator :)

### PR DESCRIPTION
**Pull request by tyrsson**
Signed-off-by: Joey Smith (Tyrsson) jsmith@webinertia.net
Changed the dash to underscore as default separator for the extra array fields. In Writers.md
Obvious Fix

|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,6 @@
 /vendor/
 .phpunit.result.cache
 .phpcs-cache
+.buildpath
+.project
+.settings/org.eclipse.php.core.prefs
```
Comment (froschdesign) [inline .gitignore]: Please add this to your local Git configuration and undo these changes here.
Thanks in advance! 👍

**Review by froschdesign — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (tyrsson) [reply .gitignore]: Yea I was using Zend studio then which made it a pain. I'll fix it this evening.

Comment (tyrsson) [reply .gitignore]: @froschdesign 
Should be fixed now.

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
Reference:

https://github.com/laminas/laminas-log/blob/91964dd3afec183c09cca5bc2b21a930a56d5237/src/Writer/Db.php#L43-L48

**Comment by froschdesign** [comment]
@Tyrsson 
Thank you for your time and this contribution! 👍

_Sorry for the delay, I was busy with client projects. Thank you for your patience!_
